utils/src/game_modes/airdash.rs

```diff
@@ -0,0 +1,73 @@
+use smash::app::{self, BattleObject, BattleObjectModuleAccessor, lua_bind::*};
+use smash::lib::lua_const::*;
+use smash::phx::{Hash40, Vector4f};
+use smash::hash40;
+use smash::lua2cpp::L2CFighterCommon;
+use crate::modules::*;
+use utils_dyn::consts::*;
+use utils_dyn::ext::*;
+use smash::phx::Vector3f;
+use crate::util;
+
+pub unsafe fn update() {
+    // skip this frame because the match hasnt started
+    if !app::sv_information::is_ready_go() {
+        return;
+    }
+
+    println!("doing airdash update!");
```

Comment:
We probably shouldn't have these printing every frame if we can help it

Reply:
great catch, thanks